### PR TITLE
Dockerfile better entrypoint

### PR DESCRIPTION
**Pull request by simonseyock**
This is a draft for solving #937

It is not fully tested.

### CODE REVIEW

**Comment by simonseyock** [comment]
This is now fully tested. The CI process is not yet adapted.

**Comment by simonseyock** [comment]
I updated the dockerbuild github action. I removed the workflow_dispatch trigger because:
a) it made the action way more complicated (double build definitions for every flavor)
b) a dockerfile should *always* correspond to a git tag. so if a tag should be republished it needs to be deleted and recreated

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+
+FROM base-libs AS builder
+
+RUN mkdir /mapproxy
+
+WORKDIR /mapproxy
+
+COPY setup.py README.md CHANGES.txt MANIFEST.in ./
+COPY mapproxy mapproxy
```
Comment (simonseyock) [inline Dockerfile:27]: These are the only files that will be included in the wheel. Maybe we need some more?

**Comment by weskamm** [comment]
> I updated the dockerbuild github action. I removed the workflow_dispatch trigger because: ...
As long as we dont need to publish older images again (which i currently see no reason for), this should be fine

File: Dockerfile (hunk 1 above)
Comment (weskamm) [reply Dockerfile:27]: License at least

**Review by weskamm — APPROVED** [review]
Thanks for your work! Lets see if tests from Hannes succeed before merging, but you get my thumbs up!

**Comment by hblitza** [comment]
Thanks @simonseyock. I tested it locally - successfully for dev,plain,nginx both alpine and debian.

**Review by hblitza — APPROVED** [review]

**Pull request merged**